README.md

```diff
@@ -70,15 +70,19 @@ You can run the formatters and linters locally. To do that:
 
 # Deployment
 
-This API is temporarily deployed on AWS Elastic Beanstalk.
-We will move it to a more stable service once the API matures.
+Deployment of this service is now done through terraform cloud.
+
+# Environmental Variables
+
+- AUTH0_DOMAIN - The Auth0 domain which can be collected from the Applications/Applications tab. It should be something like
```

Comment:
I suggest that we put all env vars in backticks, as it formats more nicely in markdown

```suggestion
- `AUTH0_DOMAIN` - The Auth0 domain which can be collected from the Applications/Applications tab. It should be something like
```